trunk/ghost/ghostparse.c

```diff
@@ -24,63 +24,7 @@ Foundation, Inc., 59 Temple Place - Suite 330, Boston, MA  02111-1307, USA.
 
 
 #define MAX_CHAINED_DEMOS   64
-
-
-/*
- * RECORD LISTS
- *
- * Records are pushed onto a linked list before being compressed into a dense
- * array.
- */
```

Comment:
There are no lists any more, just an evergrowing re-alloc'd array of records for each level.